src/provider_manifests.py

```diff
@@ -61,6 +71,7 @@ def __call__(self, obj):
         """Patch the k8s-keyston-auth deployment."""
         if not (obj.kind == "Deployment" and obj.metadata.name == RESOURCE_NAME):
             return
+        obj: Deployment = obj
```

Comment:
Why redeclare obj here?

Reply:
well -- honestly just b/c it helped my editor understand  the type.  I'm okay to remove it

Reply:
why not add the type to the parameter?

Reply:
well, reasons -- the correct type for `obj` is `AnyResource`.  Lemme see if i can make the type better...

Reply:
Ah, if the "kind" property is different from "Deployment", the type will not be Deployment. Gotcha.

Reply:
I've improved, see how ya like

Reply:
Nice ! +1